demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -356,3 +360,40 @@ def check_script_name(self, pb_script_name, id_set_scripts):
         return any(
             [pb_script_name == id_set_dict[key].get('name') for id_set_dict in id_set_scripts
              for key in id_set_dict])
+
+    def _is_id_uuid(self):
+        """
+        Check that taskid field and id field under task field are both from uuid type
+        Returns: True if the ids are uuid
+        """
+        is_valid = True
+        tasks: dict = self.current_file.get('tasks', {})
+        for task_key, task in tasks.items():
+            taskid = task.get('taskid', '')
+            inner_id = task.get('task', {}).get('id', '')
+            is_valid = is_string_uuid(taskid) and is_string_uuid(inner_id)
+
+            if not is_valid:
+                error_message, error_code = Errors.invalid_uuid(task_key, taskid, inner_id)
+                if self.handle_error(error_message, error_code, file_path=self.file_path):
+                    return is_valid
```

Comment:
it is clearer to break the loop and have a single exist(return) point in the func
```suggestion
            if not is_valid:
                error_message, error_code = Errors.invalid_uuid(task_key, taskid, inner_id)
                if self.handle_error(error_message, error_code, file_path=self.file_path):
                    break
```